src/Compatibility/Core/src/iOS/Platform.cs

```diff
@@ -655,30 +656,43 @@ internal static string ResolveMsAppDataUri(Uri uri)
 		internal void SubscribeToAlertsAndActionSheets()
 		{
 			var busyCount = 0;
-			MessagingCenter.Subscribe(this, Page.BusySetSignalName, (Page sender, bool enabled) =>
+
+			WeakReferenceMessenger.Default.Register<Platform, PageBusyMessage>(this, (platform, message) =>
```

Comment:
This is introducing a closure to capture `this` in the handler, as it's calling that `PageIsChildOfPlatform`. Why not invoke it from the recipient directly instead to avoid allocations? Same thing for the other 3 handlers defined below as well.